install-scripts/linux/skm-install-linux.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+set -eu
+GIT_SKM_REPO=https://github.com/splashkit/skm.git
+MARK_CHECK="\xE2\x9C\x94"
+MARK_CROSS="\xE2\x9C\x95"
+
+HOME_PATH=~
+INSTALL_PATH="${HOME_PATH}/.splashkit"
+
+function has_git() {
+    git --help 2>&1 > /dev/null
```

Comment:
**suggestion:** I think using `command -v` is more explicit when checking for command availability. WDYT?

```suggestion
    command -v git >/dev/null 2>&1 
```